kyuubi-common/src/main/scala/org/apache/kyuubi/Utils.scala

```diff
@@ -140,7 +140,9 @@ object Utils extends Logging {
     dir
   }
 
-  def currentUser: String = UserGroupInformation.getCurrentUser.getShortUserName
+  def currentUser: String =
+    try UserGroupInformation.getCurrentUser.getShortUserName
+    catch { case _: Exception => sys.props.get("user.name").get }
```

Comment:
Can we introduce a util method `hadoopClassPresent`? We may need this in other places.

```
private[spark] def hadoopClassPresent: Boolean = {
    try {
      Utils.classForName("org.apache.hadoop.security.UserGroupInformation")
      true
    } catch {
      case _: ClassNotFoundException | _: NoClassDefFoundError => false
    }
  }
```

`sys.props.get("user.name").get` => `sys.props("user.name")`